tests/private-registry.yaml

```diff
@@ -0,0 +1,12 @@
+---
+imageRegistry: example.com:5000
+imageName: prefix/s3gw
+imageTag: v0.6.1-private
```

Comment:
Is it possible to use a placeholder for the version here?
This could be something prone to be forgotten when releasing.

Reply:
Not necessary. This test is just to make sure the templating results in a valid yaml and the actual tag doesn't matter. I chose one that doesn't even exist but is similar enough to result in a meaningful test.